### ISSUE TITLE
[Bug]: The "Trigger UFO Abduction and trigger Pizza delivery doesnt works most of the time

### ISSUE THREAD

**Issue by cultysdaop** [Yimura/YimMenu#3395]
### Describe the bug

[Bug]: The "Trigger UFO Abduction and trigger Pizza delivery doesnt works most of the time, i click at it, it may work once for ufo abduction and after that it will not work anymore, same to pizza delivery, pls fix

### Steps To Reproduce

[Bug]: The "Trigger UFO Abduction and trigger Pizza delivery doesnt works most of the time, i click at it, it may work once for ufo abduction and after that it will not work anymore, same to pizza delivery, pls fix

### Expected Behavior

work

### Logs

[Bug]: The "Trigger UFO Abduction and trigger Pizza delivery doesnt works most of the time, i click at it, it may work once for ufo abduction and after that it will not work anymore, same to pizza delivery, pls fix

### Store

Steam

### Language

English

### YimMenu Language

English (US)

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### Coload

- [X] I've tested this without co-loading anything.

**Comment by xiaoxiao921** [comment]
IIrc that code was meant to be removed from the PR, guess it still went through

**Comment by cultysdaop** [comment]
like half of the time works the ufo abduction and sometimes pizza delivery and half of the time it does not work for me

**Comment by snowleopardd** [comment]
Sometimes it works, i think it's a fun feature, needs to be fixed not removed.

**Comment by cultysdaop** [comment]
> Sometimes it works, i think it's a fun feature, needs to be fixed not removed.
I agree, I remember when there was in the Yim Menu a feature that would sent a "Gooh" that would rob players i dont remember like 10k or something and run away, even tho it wasnt a winter anymore, but it had delayed time, just slow, it was working half of the time and half not, even if it worked it was slow and you had to wait for gooh or dont let player get in the vehicle
More fun features like that UFO Abduction and pizza delivery and add back gooh but fix it and make it faster, instant spawn

Also what does "Pizza delivery do"? I saw it there was a player next to a void and next to him a pizza scooter and that was it, i dont really see sense of using it @mikedrugs  
@DayibBaba

**Comment by ShinyWasabi** [comment]
`Global_1887305[PLAYER::PLAYER_ID() /*610*/].f_10.f_32`
This global can be used to start any freemode mission by their FMMC ID (0-340). Use value 314 for UFO abduction, and 340 for Pizza Work.

**Comment by snowleopardd** [comment]
> > Sometimes it works, i think it's a fun feature, needs to be fixed not removed.
> > I agree, I remember when there was in the Yim Menu a feature that would sent a "Gooh" that would rob players i dont remember like 10k or something and run away, even tho it wasnt a winter anymore, but it had delayed time, just slow, it was working half of the time and half not, even if it worked it was slow and you had to wait for gooh or dont let player get in the vehicle
> > More fun features like that UFO Abduction and pizza delivery and add back gooh but fix it and make it faster, instant spawn
> 
> Also what does "Pizza delivery do"? I saw it there was a player next to a void and next to him a pizza scooter and that was it, i dont really see sense of using it @mikedrugs @DayibBaba

It teleports the player to some event, they get teleported inside an alien ship

**Comment by karinaquill** [comment]
> `Global_1887305[PLAYER::PLAYER_ID() /*610*/].f_10.f_32` This global can be used to start any freemode mission by their FMMC ID (0-340). Use value 314 for UFO abduction, and 340 for Pizza Work.

Do you happen to have a list of all FHMC ids & what they correlate to handy? Googlefu failed.

**Comment by ShinyWasabi** [comment]
> Do you happen to have a list of all FHMC ids & what they correlate to handy? Googlefu failed.

Yes, I'll make a script for this.

**Comment by karinaquill** [comment]
> > Do you happen to have a list of all FHMC ids & what they correlate to handy? Googlefu failed.
> 
> Yes, I'll make a script for this.

merci :)

**Comment by cultysdaop** [comment]
> Global_1887305[PLAYER::PLAYER_ID() /*610*/].f_10.f_32

How to do that? i tried using lua script, but i failed and it dont works for me
I tried pasting that script into lue but it just doesnt paste into it
Pls someone give the proof that the ufo abduction and pizza delivery is broken most of the time it dont works for me when im clicking it
